packages/client/src/App.re

```diff
@@ -2,25 +2,31 @@
 let make = () => {
   open Shared;
 
-  let ({errorToString, _}, _changeLocale): LocaleContext.t =
-    LocaleContext.use();
+  let ({errorToString}, changeLocale): LocaleContext.t = LocaleContext.use();
+
   let url = ReasonReactRouter.useUrl();
 
   let (loggedIn, setLoggedIn) =
     React.useState(_ => AuthStorage.getLoginToken()->Belt.Option.isSome);
 
   let handleLogin = (payload: Shared.AuthPayload.t) => {
     setLoggedIn(_prev => true);
+
     AuthStorage.setLoginToken(payload.token);
+
+    changeLocale(
+      LocaleContext.SetLocale(
+        payload.language->Belt.Option.getWithDefault(`SWEDISH),
+      ),
+    );
```

Comment:
Default to browser-language if not language set? Perhaps something like:

```ocaml
payload.language->Belt.Option.isSome ? changeLocale : ()
```

or

```ocaml
Belt.Option.(
  payload.language
  ->map(userLang => changeLang(SetLocale(userLang)))
  ->getWithDefault()
);
```

Reply:
I think we explicitly said that language could be null, right?

Reply:
Yep, and if not set default to the browser-language, right? I think that'd make the most sense!

Reply:
Allright, I thought this comment was regarding what you said here: "I've noticed that there are any-values emerging in the API"
Just wanted to make sure